Makefile

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
-ENVTEST_K8S_VERSION = 1.31.0
+ENVTEST_K8S_VERSION = 1.32.0
```

Comment:
This will need to be `1.32.1`, there is no 1.32.0 in the OCP GCS bucket.
Ah we also need to set the OCP GCS bucket index still!

See: https://github.com/openshift/cluster-control-plane-machine-set-operator/blob/c7b3d7b57a8ede5e39955a223f11035e4966e293/Makefile#L94